composer.json

```diff
@@ -11,15 +11,15 @@
     ],
     "license" : "MIT",
     "require": {
-        "illuminate/support": "^5.8",
-        "illuminate/container": "^5.8",
-        "illuminate/database": "^5.8",
-        "illuminate/events": "^5.8",
+        "illuminate/support": "^6.0",
+        "illuminate/container": "^6.0",
+        "illuminate/database": "^6.0",
+        "illuminate/events": "^6.0",
         "mongodb/mongodb": "^1.4"
     },
     "require-dev": {
-        "phpunit/phpunit": "^6.0|^7.0",
-        "orchestra/testbench": "^3.1",
+        "phpunit/phpunit": "^8.0",

```

Comment:
This will probably break travis tests on older PHP versions.

Reply:
Okay, I see that it has done that already on this PR...

What would be the less breaking way of doing that?

Reply:
`"^6.0|^7.0|^8.0",` should work I think.